tests/mongo-go-driver.yml

```diff
@@ -157,6 +155,8 @@ results:
       - go.mongodb.org/mongo-driver/x/mongo/driver/integration/TestAggregate/Multiple_Batches
       - go.mongodb.org/mongo-driver/x/mongo/driver/integration/TestAggregate/TestMaxTimeMSInGetMore
     fail:
+      - go.mongodb.org/mongo-driver/mongo/integration/TestCausalConsistency_NotSupported
```

Comment:
There must be a comment explaining why those tests fail

Reply:
They were previously [passing](https://github.com/FerretDB/dance/blob/main/tests/mongo-go-driver.yml#L127-L128) now they panic.

```go.mongodb.org/mongo-driver/mongo/integration/TestCausalConsistency_NotSupported:
	=== RUN   TestCausalConsistency_NotSupported
	--- FAIL: TestCausalConsistency_NotSupported (0.08s)
	
go.mongodb.org/mongo-driver/mongo/integration/TestCausalConsistency_NotSupported/afterClusterTime_not_included:
	=== RUN   TestCausalConsistency_NotSupported/afterClusterTime_not_included
	    --- FAIL: TestCausalConsistency_NotSupported/afterClusterTime_not_included (0.08s)
	panic: runtime error: invalid memory address or nil pointer dereference [recovered]
		panic: runtime error: invalid memory address or nil pointer dereference
	[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xe47b76]
	
	goroutine 72 [running]:
	testing.tRunner.func1.2({0xecf8e0, 0x17877e0})
		/usr/local/go/src/testing/testing.go:1526 +0x372
	testing.tRunner.func1()
		/usr/local/go/src/testing/testing.go:1529 +0x650
	panic({0xecf8e0, 0x17877e0})
		/usr/local/go/src/runtime/panic.go:890 +0x263
	go.mongodb.org/mongo-driver/mongo/integration.TestCausalConsistency_NotSupported.func1(0xc0002f0c40)
		/home/byron/workdir/dance/tests/mongo-go-driver/mongo/integration/causal_consistency_test.go:225 +0x256
	go.mongodb.org/mongo-driver/mongo/integration/mtest.(*T).RunOpts.func1(0x0?)
		/home/byron/workdir/dance/tests/mongo-go-driver/mongo/integration/mtest/mongotest.go:266 +0x5a9
	testing.tRunner(0xc0002fbba0, 0xc00003cde0)
		/usr/local/go/src/testing/testing.go:1576 +0x217
	created by testing.(*T).Run
		/usr/local/go/src/testing/testing.go:1629 +0x806
```

Reply:
I am not super confident on this but it seems that the changes [made](https://github.com/mongodb/mongo-go-driver/compare/6d814184973ae7ffc13281c93b423dc522a4bcca...6c7e1241b747d2fb5d164c9ca9d17617f33e7899#diff-9d480eb84448ec1035a67cc3d37fd15d2aa1eb958ea6f8fb0d1c3b0d8ac85b27L226) now cause their event monitor to return nil [here](https://github.com/mongodb/mongo-go-driver/blob/67f257ac1d970452575fa24d7394e1ed46d1d725/mongo/integration/causal_consistency_test.go#L224) when it tries to inspect the command name. Could be related to https://github.com/mongodb/mongo-go-driver/pull/1347?

@AlekSi should I add this failing test to common ignore for now?

Reply:
But it only fails on FerretDB which invalidates the above.